doma-core/src/main/java/org/seasar/doma/jdbc/criteria/statement/EntityqlMultiInsertStatement.java

```diff
@@ -35,6 +37,26 @@ public EntityqlMultiInsertStatement(
     this.settings = Objects.requireNonNull(settings);
   }
 
+  /**
+   * Create statement that inserts or updates
+   *
+   * @return statement
+   */
+  public EntityqlMultiInsertStatement<ENTITY> onDuplicateKeyUpdate() {
```

Comment:
return type was maybe
`Statement<MultiResult<ENTITY>>` is preferred.
because, don't want to allow following code.

```java
// can call Call multiple times onDuplicateKeyUpdate() method
entityql.insertMulti(d, departments).onDuplicateKeyUpdate().onDuplicateKeyUpdate().execute();
```